src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -81,6 +108,24 @@
  *       value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void equals() { // violation, not be annotated with &#64;Override
+ *
+ *     }
+ * }
+ *
+ * class Test extends SuperClass {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test() { // OK, only checks methods overridden from java.lang.Object
```

Comment:
`only checks methods overridden from java.lang.Object`
It is not clear. 
This is ignored because class extends other class

Reply:
ok, I correct this.